docs/uenv-linaro-forge.md

```diff
@@ -27,19 +27,19 @@ they can be used alongside application and development uenv mounted at
     When using alongside another uenv, start a uenv session with both uenv with `linaro-forge` after the main uenv, to mount the images at the respective `/user-environment` and `/user-tools` locations:
 
     ```bash
-    uenv start prgenv-gnu/24.2:v3 linaro-forge/23.1.2
+    uenv start --view=prgenv-gnu:default prgenv-gnu/24.2:v3 linaro-forge/23.1.2
 
     # test that everything has been mounted correctly
     # (will give warnings if there are problems)
     uenv status
 
     source /user-tools/activate
-    uenv view prgenv-gnu:default
 
     # check that ddt is in the path
     ddt --version
     ```
 
+    TODO: Does this break with uenv start --view?
     The `/user-tools/activate` script will make the forge executables available in your environment, and **must be run before**
     any other uenv view command otherwise the environment variables set from the other uenv will be overwritten.
```

Comment:
@jgphpc could you clarify what the requirements are here? Is the `/user-tools/activate` script not going to work correctly with the view loaded at `start`? Is it a requirement to source the `activate` script?

Reply:
At the time of doc writing, this was the only way but now, loading 2 views is working fine:
```
uenv start prgenv-gnu/24.11:v1,linaro-forge/24.0.2:v1 -v forge,prgenv-gnu:default
uenv start prgenv-gnu/24.11:v1,e0e79f5c3e6a8ee0 -v forge,prgenv-gnu:default

which nvcc
/user-environment/env/default/bin/nvcc

which ddt
/user-tools/env/forge/bin/ddt
```
so it's fine to remove that part.

Reply:
Thanks @jgphpc! I've updated this to load both uenvs and views in one command. Please have a look to see if it's sane to you.